keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -404,6 +490,11 @@ The following scenarios are covered:
 - the `job_by_external_controller_total` metric is not incremented for a new Job without "managedBy" or with default value
 - the `job_by_external_controller_total` metric is not incremented for Job updates (regardless of the "managedBy")
 
+The following scenarios related to [Terminating pods and terminal Job conditions](#terminating-pods-and-terminal-job-conditions) are covered:
+- `Failed` or `Complete` conditions are not added while there are still terminating pods
+- `FailureTarget` is added when backoffLimitCount is exceeded, or activeDeadlineSeconds timeout is exceeded
+- `SuccessCriteriaMet` is added when the `completions` are satisfied
```

Comment:
I have a question about responsibility. This change is needed to graduate the SuccessPolicy (KEP3998)? Or we can graduate the SuccesPolicy regardless of this change?

Reply:
IMO we can graduate independently. The new fix reuses the already introduced conditions which match the purpose.

Reply:
That makes sense. 
The `SuccessCriteriaMet` condition is still guarded by the SuccessPolicy feature gate, right?
I meant the new transition is not guarded by the ManagedBy feature gate, right?

Reply:
Depends on the context. When adding it for meeting success policy we guard it by the success policy feature gate. When adding for meeting completions we guard it by managedBy or podreplacementpplicy feature gates. The code which translates SuccessCriteriaMet into Complete condition will be guarded by OR of the feature gates.

Reply:
That makes sense. Maybe we can revisit here once we graduate the successPolicy to GA.